modules/openy_traction_rec_import/src/Commands/OpenyTractionRecImportCommands.php

```diff
@@ -109,31 +97,25 @@ public function __construct(
    */
   public function import(array $options): bool {
     if (!$this->importer->isEnabled()) {
-      $this->logger()->notice(
-        dt('Traction Rec import is not enabled!')
-      );
+      $this->logger()->notice('Traction Rec import is not enabled!');
       return FALSE;
     }
 
     if (!$this->importer->acquireLock()) {
-      $this->logger()->notice(
-        dt('Can\'t run new import, another import process already in progress.')
-      );
+      $this->logger()->notice('Can\'t run new import, another import process already in progress.');
       return FALSE;
     }
 
     if (!$this->importer->checkMigrationsStatus()) {
-      $this->logger()->notice(
-        dt('One or more migrations are still running or stuck.')
-      );
+      $this->logger()->notice('One or more migrations are still running or stuck.');
       return FALSE;
     }
 
     $this->output()->writeln('Starting Traction Rec migration');
 
     $dirs = $this->importer->getJsonDirectoriesList();
     if (empty($dirs)) {
-      $this->logger()->info(dt('Nothing to import.'));
+      $this->logger()->info('Nothing to import.');
```

Comment:
We need to define a context
```suggestion
      $this->logger()->info('Traction REC: Nothing to import.');
```